app/views/includes/header.nunjucks

```diff
@@ -0,0 +1,53 @@
+<header class="site-header">
+  <div class="site-header__top">
+    <div class="site-header__brand">
+      <!-- noindex -->
+      <a href="/">
+        <img
+          src="{{ SITE_ROOT }}/images/choices-logo.png" srcset="
+              {{ SITE_ROOT }}/images/choices-logo.png 1x,
+              {{ SITE_ROOT }}/images/choices-logo@2x.png 2x,
+              {{ SITE_ROOT }}/images/choices-logo@3x.png 3x,"
+          class="nhs-choices-logo"
+        />
+      </a>
+      <span class="site-header__tagline">Your health, your choices</span>
+    </div>
+
+    <div class="site-header__controls">
+      <div class="site-header__user-links user-links">
+        <a class="user-links__translate" href="http://www.nhs.uk/pages/languagehub.aspx">Translate</a>
+        <span class="user-links__right-align">
+          <a class="user-links__account" href="https://www.nhs.uk/Personalisation/MyNHSChoices.aspx">User Accounts</a>
+        </span>
+      </div>
+      <div id="search" class="site-header__search search">
+        <form id="gs" method="get" action="https://www.nhs.uk/Search/">
+          <input name="q" class="search__input" type="text" placeholder="Search" />
+          <div class="search__button icon-Search">
+            <input type="submit" value=""/>
+          </div>
+        </form>
+      </div>
+      <div id="menu-icon" class="site-header__menu-icon menu-icon">
+          <a id="main-nav-toggle" class="menu-icon__link" href="#">Menu</a>
+      </div>
+    </div>
+
+    <div class="clearfix"></div>
+  </div>
+
+  <nav id="main-nav" class="site-header__nav main-nav main-nav">
+
```

Comment:
Can this blank line be deleted if it isn't required?

Reply:
Gone